third_party/terraform/data_sources/data_source_google_sql_ca_certs.go

```diff
@@ -0,0 +1,92 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceGoogleSQLCaCerts() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleSQLCaCertsRead,
+
+		Schema: map[string]*schema.Schema{
+			"instance": {
+				Type:             schema.TypeString,
+				Required:         true,
+				DiffSuppressFunc: compareSelfLinkOrResourceName,
+			},
+			"project": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+				ForceNew: true,
+			},
+			"active_version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"certs": {
+				Type: schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"cert": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"common_name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"create_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"expiration_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"sha1_fingerprint": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceGoogleSQLCaCertsRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	var project, instance string
+	if v, ok := d.GetOk("instance"); ok {
```

Comment:
nit: instance is required, so we can remove this if/else and just keep the logic that's inside

Reply:
Good point. Removed